.travis.yml

```diff
@@ -4,16 +4,5 @@ matrix:
       python: 3.6
       before_install:
         - export CI
-      install:
-        - pip install -U platformio
-        - platformio update
-        - wget https://github.com/pathfinder-for-autonomous-navigation/FlightSoftware/releases/download/v0.11/linux-x86_64_hootl
-        - cp linux-x86_64_hootl python/linux-x86_64_hootl
-        - cd python
-        - chmod +x linux-x86_64_hootl
-        - pip install -r requirements.txt
       script:
-        - python -m pytest
-        #- python -m ptest.run_simulation
-        - cd ..
-        - platformio test --verbose --environment ci_github
```

Comment:
Don't we still want to leave the platformio tests for GNC code? Don't see any reason to get rid of those as well.

Reply:
Oops, thanks, I didn't realize those are still in.